core/templates/pages/exploration-editor-page/feedback-tab/services/thread-data.service.ts

```diff
@@ -16,26 +16,31 @@
  * @fileoverview Service for getting thread data from the backend for the
  * feedback tab of the exploration editor.
  */
-
-require('domain/feedback_message/ThreadMessageObjectFactory.ts');
-require('domain/feedback_thread/FeedbackThreadObjectFactory.ts');
-require('domain/suggestion/SuggestionThreadObjectFactory.ts');
-require('domain/utilities/url-interpolation.service.ts');
-require(
-  'pages/exploration-editor-page/exploration-editor-page.constants.ajs.ts');
-require('pages/exploration-editor-page/exploration-editor-page.constants.ts');
-require('services/alerts.service.ts');
-require('services/context.service.ts');
-require('services/suggestions.service.ts');
-
-import { FeedbackThread, FeedbackThreadBackendDict } from
-  'domain/feedback_thread/FeedbackThreadObjectFactory';
-import { SuggestionThread } from
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+// Import cloneDeep from 'lodash/cloneDeep';
+import {FeedbackThread, FeedbackThreadBackendDict, FeedbackThreadObjectFactory }
+  from 'domain/feedback_thread/FeedbackThreadObjectFactory';
+import { SuggestionThread, SuggestionThreadObjectFactory } from
   'domain/suggestion/SuggestionThreadObjectFactory';
-import { ThreadMessage } from
+import { ThreadMessage, ThreadMessageObjectFactory } from
   'domain/feedback_message/ThreadMessageObjectFactory';
 import { SuggestionBackendDict } from
   'domain/suggestion/SuggestionObjectFactory';
+import { HttpClient } from '@angular/common/http';
+import { AlertsService } from 'services/alerts.service.ts';
+import { ContextService } from 'services/context.service.ts';
+import { SuggestionsService } from 'services/suggestions.service.ts';
+import { UrlInterpolationService }
+  from 'domain/utilities/url-interpolation.service.ts';
+import { ExplorationEditorPageConstants } from
+  'pages/exploration-editor-page/exploration-editor-page.constants';
+
+const ACTION_ACCEPT_SUGGESTION =
+ExplorationEditorPageConstants.ACTION_ACCEPT_SUGGESTION;
```

Comment:
should this be indented by 2?

Reply:
My bad, will make appropriate changes to this